src/pages/workspaces/workspace/useWorkspace.test.ts

```diff
@@ -3,17 +3,24 @@ import _ from 'lodash/fp'
 import { locationTypes } from 'src/components/region-common'
 import * as WorkspaceUtils from 'src/components/workspace-utils'
 import { Ajax } from 'src/libs/ajax'
+import { AzureStorage, AzureStorageContract } from 'src/libs/ajax/AzureStorage'
 import * as GoogleStorage from 'src/libs/ajax/GoogleStorage'
 import { getConfig } from 'src/libs/config'
 import * as Notifications from 'src/libs/notifications'
 import { getUser, workspaceStore } from 'src/libs/state'
+import { DeepPartial } from 'src/libs/type-utils/deep-partial'
 import { defaultLocation } from 'src/pages/workspaces/workspace/analysis/runtime-utils'
 import {
   azureBucketRecheckRate, googlePermissionsRecheckRate, useWorkspace
 } from 'src/pages/workspaces/workspace/useWorkspace'
+import { asMockedFn } from 'src/testing/test-utils'
 
 
+jest.mock('src/libs/ajax/AzureStorage')
+
 jest.mock('src/libs/ajax')
+type AjaxExports = typeof import('src/libs/ajax')
```

Comment:
the convention for this is to put the MyModuleExports type just above the jest.mock call on that same module so that it is consistent, and available for an inline mock definition.

Reply:
I'm not clear… you are just suggesting that I move line 22 above line 21?

Reply:
And what about line 23? 

(I'm confused because it doesn't seem to matter what order these are in)

Reply:
Fixed in https://github.com/DataBiosphere/terra-ui/pull/3765/commits/b32f2e67a05ae7f65b61fdd8d2061f56493a8a3f

Reply:
sorry, just seeing this now.... Yes define AjaxExports just above line 21, and probably have line 23 be moved just before AjaxExports new spot.  

AjaxExports move is to align with unit test guide examples.  Having the export type be right above the jest.mock call makes it easier to eyeball that the import paths match.  

I hope to promote the AjaxContract type to a shared spot to just let various modules/test-files reference it.  I am still beefing up the Typescript contributors guide.  The cost of getting in early on the TS train :)

created a ticket:
https://broadworkbench.atlassian.net/browse/UIE-69